newsfragments/1660.bugfix.rst

```diff
@@ -1 +1 @@
-- Added formatter rules for eth_tester middleware to allow getBalance by using integer block numbers
\ No newline at end of file
+Added formatter rules for eth_tester middleware to allow :meth:`~web3.eth.Eth.getBalance` by using integer block numbers
```

Comment:
FWIW I often make mistakes on the exact ReST syntax. Definitely something worth watching for during release time. I like to build the docs locally to see if the links work (after running `make notes`, but before `make release`).